Enhance E3 mini V3 config

clee opened the pull request:
- Link to the Klipper thermistor page instead of hardcoding a list that might be outdated as comments
- Comment out `sensor_type` lines to force people to pick
- Add FAN2 config

FHeilmann commented:
While you're fixing up this config w.r.t. thermistors, could you please move the thermistor definition of `Trianglelab NTC100K B3950` above the `[extruder]` block? If users choose this thermistor with the config as is, the thermistor will be referenced before it is defined, causing errors.

Relevant section of the klipper docs:
```
Custom thermistors (one may define any number of sections with a "thermistor" prefix). 
A custom thermistor may be used in the sensor_type field of a heater config section.
(For example, if one defines a "[thermistor my_thermistor]" section then one may 
use a "sensor_type: my_thermistor" when defining a heater.) Be sure to 
place the thermistor section in the config file above its first use in a heater section.
```

clee commented:
> While you're fixing up this config w.r.t. thermistors, could you please move the thermistor definition of `Trianglelab NTC100K B3950` above the `[extruder]` block? If users choose this thermistor with the config as is, the thermistor will be referenced before it is defined, causing errors.

Good call.

clee committed:
Enhance E3 mini V3 config

- Link to the Klipper thermistor page instead of hardcoding a list that
  might be outdated  as comments
- Add FAN2 config
